build.gradle

```diff
@@ -22,23 +28,32 @@ targetCompatibility = 1.8
 repositories {
     mavenCentral()
     maven {
-        url "http://artifactory.terasology.org/artifactory/virtual-repo-live"
+        def repoViaEnv = System.getenv()["RESOLUTION_REPO"]
+        if (rootProject.hasProperty("alternativeResolutionRepo")) {
+            // If the user supplies an alternative repo via gradle.properties then use that
+            name "from alternativeResolutionRepo property"
+            url alternativeResolutionRepo
+        } else if (repoViaEnv != null && repoViaEnv != "") {
+            name "from \$RESOLUTION_REPO"
+            url = repoViaEnv
+        } else {
+            // Our default is the main virtual repo containing everything except repos for testing Artifactory itself
+            name "Terasology Artifactory"
+            url "http://artifactory.terasology.org/artifactory/virtual-repo-live"
+            allowInsecureProtocol true  // 😱
+        }
     }
 }
 
-task wrapper(type: Wrapper) {
-    gradleVersion = '2.5'
-}
-
 configurations {
     codeMetrics
 }
 
 dependencies {
 
-    codeMetrics(group: 'org.terasology.config', name: 'codemetrics', version: '1.0.0', ext: 'zip')
+    codeMetrics(group: 'org.terasology.config', name: 'codemetrics', version: '1.3.2', ext: 'zip')
```

Comment:
https://github.com/MovingBlocks/TeraConfig was recently released as v1.5.0, containing a fix for the license header regex. If we're doing an upgrade here, let's go for 1.5.0 😉

Reply:
Did this in some minor changes I'll add to a new branch 👍